python-avd/pyavd/_cv/workflows/models.py

```diff
@@ -64,6 +65,27 @@ class CVDeviceTag:
     value: str
     device: CVDevice | None = None
 
+    def to_cv_tag(self) -> CVTag:
```

Comment:
Same everywhere where we transform the object to a different type.
```suggestion
    def as_cv_tag(self) -> CVTag:
```

Reply:
Fixed.